modules/iform_licences/iform_licences.module

```diff
@@ -57,8 +57,7 @@ function iform_licences_form_alter(&$form, &$form_state, $form_id) {
         $thisLicenceId = $licenceFor === 'records' ? $currentLicenceId : $currentMediaLicenceId;
         $description = t('Select the licence to apply to your @label.', ['@label' => $label]) . ' ';
         // For media only, add some text to recommend CC0 and avoiding commercially valuable images
-        $description .= $licenceFor === 'records' ? '' : t('<b>If you are happy to select a CC0 or CC BY licence option, that will provide the best chance for your photos to be used in research and conservation.</b> ');
-
+        $description .= $licenceFor === 'records' ? '' : t('If you are happy to select a CC0 or CC BY licence option, that will provide the best chance for your photos to be used in research and conservation. ');
```

Comment:
I think this would read better as a simple if statement:
```php
if ($licenceFor !== 'records') {
  $description  .= ...;
}

Reply:
Updated.